spec/test_data/v4.2/expected_wand.rbi

```diff
@@ -1,199 +1,199 @@
 # This is an autogenerated file for dynamic methods in Wand
-# Please rerun rake rails_rbi:models to regenerate.
+# Please rerun rake rails_rbi:models[Wand] to regenerate.
+
 # typed: strong
+module Wand::EnumInstanceMethods
+  extend T::Sig
 
-class Wand::ActiveRecord_Relation < ActiveRecord::Relation
-  include Wand::ModelRelationShared
-  extend T::Generic
-  Elem = type_member(fixed: Wand)
-end
+  sig { returns(T::Boolean) }
+  def phoenix_feather?; end
 
-class Wand::ActiveRecord_Associations_CollectionProxy < ActiveRecord::Associations::CollectionProxy
-  include Wand::ModelRelationShared
-  extend T::Generic
-  Elem = type_member(fixed: Wand)
-end
+  sig { void }
+  def phoenix_feather!; end
 
-class Wand < ApplicationRecord
-  extend T::Sig
-  extend T::Generic
-  extend Wand::ModelRelationShared
-  include Wand::InstanceMethods
-  Elem = type_template(fixed: Wand)
-end
+  sig { returns(T::Boolean) }
+  def dragon_heartstring?; end
 
-module Wand::InstanceMethods
-  extend T::Sig
+  sig { void }
+  def dragon_heartstring!; end
+
+  sig { returns(T::Boolean) }
+  def unicorn_tail_hair?; end
 
   sig { void }
-  def basilisk_horn!(); end
+  def unicorn_tail_hair!; end
 
   sig { returns(T::Boolean) }
-  def basilisk_horn?(); end
+  def basilisk_horn?; end
+
+  sig { void }
+  def basilisk_horn!; end
+end
+
+module Wand::GeneratedAttributeMethods
+  extend T::Sig
 
   sig { returns(T::Boolean) }
-  def broken(); end
+  def broken; end
 
   sig { params(value: T::Boolean).void }
   def broken=(value); end
 
-  sig { params(args: T.untyped).returns(T::Boolean) }
-  def broken?(*args); end
+  sig { returns(T::Boolean) }
+  def broken?; end
 
   sig { returns(T.nilable(Date)) }
-  def chosen_at_date(); end
+  def chosen_at_date; end
 
   sig { params(value: T.nilable(Date)).void }
   def chosen_at_date=(value); end
 
-  sig { params(args: T.untyped).returns(T::Boolean) }
-  def chosen_at_date?(*args); end
```

Comment:
Was removing all these `args` parameters intentional?

Reply:
This is intentional. You actually cannot call these checker methods with any arg. Ruby Reflection says that there is an arg, but at run time it throws error (see below)

At first sorbet-rails try to match data from ruby reflection. After this, I gave up on matching, because it can be incorrect.

This may cause conflict with sorbet's hidden rbi generation because that's based on ruby run-runtime. It may now expect people to run `srb rbi hidden-definition` again to remove the conflicts.

```
irb(main):003:0> s.name?
=> true
irb(main):005:0> s.name?('1')
Traceback (most recent call last):
       14: from bin/rails:4:in `<main>'
       13: from bin/rails:4:in `require'
       12: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/commands.rb:18:in `<top (required)>'
       11: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/command.rb:46:in `invoke'
       10: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/command/base.rb:65:in `perform'
        9: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
        8: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
        7: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
        6: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/commands/console/console_command.rb:96:in `perform'
        5: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/commands/console/console_command.rb:19:in `start'
        4: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/railties-5.2.2.1/lib/rails/commands/console/console_command.rb:64:in `start'
        3: from (irb):5
        2: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activemodel-5.2.2.1/lib/active_model/attribute_methods.rb:382:in `name?'
        1: from /home/dev/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/activerecord-5.2.2.1/lib/active_record/attribute_methods/query.rb:37:in `attribute?'
ArgumentError (wrong number of arguments (given 2, expected 1))
```